packages/documentation/src/pages/getting-started/common-tasks/workflow/deploy.mdx

```diff
@@ -0,0 +1,90 @@
+# Deploy
+
+Code goes through several steps to get to production. This document describes this process. It should also be noted that this is the same flow for both content and code changes.
+
+## Automated Deployment Schedule
+
+| Application | Branch | Changes in by | Deployment Start | Release History |
+| --- | --- | --- | --- | --- |
+| vets-website | master | 2:00pm ET M-F | 3:00pm ET M-F| https://github.com/department-of-veterans-affairs/vets-website/releases |
+| vets-api | master | 2:00pm ET M-F | 3:00pm ET M-F | https://github.com/department-of-veterans-affairs/vets-api/releases |
+
+## Overview
+
+Jenkins performs the following tasks after a pull request is merged into `master`
+
+1. **Build** `master` branch to create an deployment artifact (.tar file)
+1. **Deploy** the artifact to dev.va.gov and staging.va.gov
+1. [Create a release](https://help.github.com/en/articles/creating-releases) in GitHub from master, tag artifacts of that commit SHA hash with release name
+1. **Deploy** to production using deployment artifact according to automated deployment schedule
+
+_A big assumption in this process is that the `master` should always be deployable. As such, the deployment to the staging environment is configured to happen automatically and can be used to see what something would look like in a production-like environment for any kind of manual testing/verification._
+
+## Process details
+
+- [Create a release](https://help.github.com/en/articles/creating-releases) in GitHub from master, tag artifacts of that commit SHA hash with release name
+
+Every work day at the configured time a [Jenkins automerge job](http://jenkins.vetsgov-internal/job/deploys/job/vets-gov-automerge/) sends a link to the #vetsgov-engineers Slack channel with a diff between the last release and the most recent changes in `master`. This commit reference is stored to ensure the diff and released version is deterministic.
+
+After a time has elapsed ( currently set to 60m ) release is created at the reference from above.
+
+- **Deploy** to production using deployment artifacts
+
+From here, Jenkins can kickoff a production deployment. After the deployment occurs, the normal site monitoring infrastructure will be used to valide it is working. As this process is automatic any new features should have monitoring in place before, or as a part of their deployment
+
+
+## Rollbacks
+
+If a production deployment introduces issues that affect Service Level Objectives (SLOs) established for the project, the DevOps team may restore service to users by rolling back the deployment. This is accomplished by triggering a new deploy job in Jenkins using a previous release tag.
```

Comment:
Didn't really edit this- we may want to formalize this into a list of things that need to happen when rolling something back breaking it up into
- criteria for a rollback
- types of rollbacks (e.g. this doesn't really suggest pushing current master with fixes- just rolling back)

Reply:
https://github.com/department-of-veterans-affairs/vets.gov-team/issues/18141